supervisord.conf

```diff
@@ -0,0 +1,11 @@
+[supervisord]
+nodaemon=true
+
+[program:vogon]
+command=/vogon/startup.sh
```

Comment:
this does not seem right. That's the dev startup script, right? So it will still run the dev server. Instead Django should be run with Guniconr and supervisor would start gunicorn. It should look somewhat like this probably:
https://github.com/diging/amphora-docker/tree/main/scripts
(without the celery part and there might be updates to it). Maybe google gunicorn, django, and supervisor and see if you can find an update ot date example.